core/src/main/java/com/yahoo/ycsb/DB.java

```diff
@@ -41,97 +41,94 @@
  * target application.  For the sake of comparison between experiments we also 
  * recommend you explain the semantics you chose when presenting performance results.
  */
-public abstract class DB
-{
-	/**
-	 * Properties for configuring this DB.
-	 */
-	Properties _p=new Properties();
+public abstract class DB {
+  /**
+   * Properties for configuring this DB.
+   */
+  private Properties properties = new Properties();
```

Comment:
Hmm, yeah I guess since none of the existing DB's use it directly and it's a tiny API change it'd be fine. We should just make a big note of it in the release notes.